packages/dashboard/src/components/tests/tooltip.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
If we use enable on the `react-components` package we should move those tests there